multiversion/src/main/scala/multiversion/outputs/DepsOutput.scala

```diff
@@ -77,4 +78,22 @@ def jvm_deps():
     jvm_deps_rule(name="maven")
 """.stripMargin
   }
+
+  def writeManifests(root: Path): Unit =
+    allTargets.foreach { target =>
+      val manifest = manifestPath(root, target)
+      val dependencies = for {
+        config <- index.thirdparty.depsByTargets(target)
+        dependency <- index.dependencies.getOrElse(config.id, Nil)
+      } yield index.reconciledDependency(dependency).repr
+      val output = dependencies.distinct.sorted.mkString(System.lineSeparator())
+      Files.createDirectories(manifest.getParent())
+      Files.write(manifest, output.getBytes)
```

Comment:
This probably needs a header showing this is a generated file.

Reply:
In phabricator the @generated header collapses the file by default in the phab.